src/Microsoft.Xaml.Behaviors/Microsoft.Xaml.Behaviors.csproj

```diff
@@ -103,7 +103,7 @@
   <Target Name="IncludeProjectToProjectAssets">
     <ItemGroup>
       <BuildOutputInPackage Include="$(ProjectDir)bin\$(Configuration)\net45\Design\Microsoft.Xaml.Behaviors.DesignTools.dll"
-                            Condition="'$(TargetFramework)' == 'netcoreapp3.0'">
+                            Condition="'$(TargetFramework.StartsWith('netcoreapp'))">
```

Comment:
Frankly I'd prefer to test both frameworks or at least test for "netcoreapp3." prefix.

Reply:
@pedrolamas I changed it to `netcoreapp3.`